src/MizarData/MizarDataTest.cpp

```diff
@@ -130,4 +136,40 @@ TEST(MizarDataTest, GetFunctionNameFromAddressIsCorrect) {
   EXPECT_FALSE(data.GetFunctionNameFromAddress(kUnknownFunctionAddress));
 }
 
+class MockMizarData : public MizarData {
+ public:
+  MOCK_METHOD(std::optional<std::string>, GetFunctionNameFromAddress, (uint64_t), (const override));
+};
+
+const uint64_t kTime = 951753;
+const orbit_client_data::CallstackInfo kCallstackInfo({kFunctionAddress, kUnknownFunctionAddress,
+                                                       kFunctionAddress},
+                                                      orbit_client_data::CallstackType::kComplete);
+constexpr uint64_t kCallstackId = 0xCA11;
+static const orbit_client_data::CallstackEvent kCallstackEvent(kTime, kCallstackId, kTID);
+static const absl::flat_hash_map<uint64_t, std::string> kSymbolsTable = {
+    {kFunctionAddress, kFunctionName}, {kAnotherFunctionAddress, "food()"}};
+
+using ::testing::Invoke;
+using ::testing::UnorderedElementsAreArray;
+
+TEST(MizarDataTest, AllAddressToNameIsCorrect) {
+  MockMizarData data;
+
+  EXPECT_CALL(data, GetFunctionNameFromAddress)
+      .WillRepeatedly(Invoke([](uint64_t addr) -> std::optional<std::string> {
+        if (const auto it = kSymbolsTable.find(addr); it != kSymbolsTable.end()) {
+          return it->second;
+        }
+        return std::nullopt;
+      }));
+
+  CallOnCaptureStarted(data);
+  data.OnUniqueCallstack(kCallstackId, kCallstackInfo);
+  data.OnCallstackEvent(kCallstackEvent);
+  data.OnCaptureFinished({});
+
+  EXPECT_THAT(data.AllAddressToName(),
+              UnorderedElementsAreArray({make_pair(kFunctionAddress, kFunctionName)}));
```

Comment:
nit: You can use `UnorderedElementsAre(std::make_pair())` (no need for the "Array")
nit: please use the std namespace explicitely `std::make_pair`